### PR TITLE
Fix/508 compliance 2

### PR DESCRIPTION
**Pull request by ZakirG**
Jira Tickets: 
Keyboard navigability of expandible/collapsible items: 
[PXP-8001](https://ctds-planx.atlassian.net/browse/PXP-8001)

Expand/collapse state defined with aria attributes: 
[PXP-7999](https://ctds-planx.atlassian.net/browse/PXP-7999)

Roles for non-button-tag buttons properly defined: 
[PXP-7998](https://ctds-planx.atlassian.net/browse/PXP-7998)

Method to bypass blocks of repetitive content: 
[PXP-7995](https://ctds-planx.atlassian.net/browse/PXP-7995)

Focus outlines on clickable elements: 
[PXP-7994](https://ctds-planx.atlassian.net/browse/PXP-7994)

Fix all HTML validator errors:
 [PXP-7992](https://ctds-planx.atlassian.net/browse/PXP-7992)

Text resizability:
[PXP-7990](https://ctds-planx.atlassian.net/browse/PXP-7990)

This PR wraps up the 508 compliance task set, making the site more accessible.

### New Features
- The Data Explorer and Discovery page now have accessibility links at the top of the pages that allow for more accessible keyboard navigation, bypassing blocks of repetitive content to tab over to areas of interest.

### Improvements
- Text can now be resized using browser text size settings.
- aria-selected and aria-expanded roles are filled in and maintained across the site for improved accessibility.
- Focus outlines are now visible on all clickable elements.
- Resolved keyboard trap issue in the Query page code editor. Users can unfocus from a textarea by pressing the Esc key.
- Page titles are now correctly populated using the gitops.json `appName` attribute.

### CODE REVIEW

File: src/components/filters/Chip/Chip.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .g3-chip {
   box-sizing: border-box;
-  font-size: 14px;
+  font-size: 1em;
```
Comment (mfshao) [inline src/components/filters/Chip/Chip.css]: why not `0.875rem` for this one?

**Review by mfshao — COMMENTED** [review]
nothing major, but there are a couple of files that don't have a new line at the end, which should be auto-fixed by the precommit hooks

check if you have precommit hooks installed for this repo?

File: src/components/filters/Chip/Chip.css (hunk 1 above)
Comment (ZakirG) [reply src/components/filters/Chip/Chip.css]: true, that's a mistake

Comment (ZakirG) [reply src/components/filters/Chip/Chip.css]: and i don't have precommit hooks installed for this repo, i'll add them

Comment (ZakirG) [reply src/components/filters/Chip/Chip.css]: ok installed precommit hooks. including proof for posterity's sake lol 
Screen Shot 2021-06-25 at 8 54 35 AM

**Review by mfshao — APPROVED** [review]
LGTM! lets update the package version

**Pull request merged**